aws/cloudformation/ami-manager.js

```diff
@@ -0,0 +1,112 @@
+var response = require('cfn-response');
+/** Takes an AWS CloudFormation stack name and instance id and returns the newly-created AMI ID. **/
+exports.handler = function (event, context) {
+  console.log("REQUEST RECEIVED:\n", JSON.stringify(event));
+
+  var stackName = event.ResourceProperties.StackName;
+  var instanceId = event.ResourceProperties.InstanceId;
+  var instanceRegion = event.ResourceProperties.Region;
+
+  var responseStatus = "FAILED";
```

Comment:
Does the responseStatus need to be hoisted here and mutated?

It only every seems to be used as an argument to response.send (with "FAILED" in one case and "SUCCESS in the other), so it seems like it would be clearer to use the literal strings in the call (or assign a constant right before the call if it's important the parameter have a meaningful name.